integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-docker-build-and-deploy-deployment/verify.groovy

```diff
@@ -15,5 +15,5 @@ kubernetesYml.withInputStream { stream ->
 
     //Check that ti contains a Deployment named after the project
```

Comment:
I see why you asked about the integration tests: 
- standard
- maven-invoker ones

The maven invoker ones require an actual cluster to run, as the perform  a real deployment. This is not currently possible when validating pull requests and this is why we use: `invoker.buildResult = failure`.

For your local tests make sure you comment out this line or change the value to `success`.

Reply:
Thank you for this hint.
My question about the reliability was more regarding that there are only little validations about the generated manifest.
But the deserialization already implies, that it's valid in general.